### ISSUE TITLE
Tracing=enabled causes logs to appear in the coordinator everytime it pings the worker.

### ISSUE THREAD

**Issue by haverma** [batfish/batfish#952]
So Allinone cannot be used with the Client as logs keep on appearing on the prompt

**Comment by dhalperi** [comment]
The issue does not seem to be logging itself. but the client re-registering the feature repeatedly?

**Comment by dhalperi** [comment]
Workaround is to disable tracing or run client separately from coordinator.

**Comment by haverma** [comment]
@dhalperi yes, the registering is done once per httpclient but a new httpclient is created every-time the coordinator refreshes worker's status, causing opentracing to put a log for every new httpclient

**Comment by dhalperi** [comment]
Fixing this week

**Comment by haverma** [comment]
Did a workaround fix in [1447](https://github.com/batfish/batfish/pull/1447) by not registering tracing for polling type http requests. 
Ideal fix would be to bump up the version of `opentracing-jaxrs`, since this message is now associated with `FINE` loglevel: ([pr](https://github.com/opentracing-contrib/java-jaxrs/pull/86)). 

But bumping up version will need some more changes since `opentracing` version that we are using is not compatible with the latest `opentracing-jaxrs` release, and also upgrading our `opentracing` api jar will need some code changes since there has been some refactoring in the `Span/ActiveSpan` hierarchy ([more info](https://github.com/opentracing/opentracing-java/commit/85433fce92575e31e362e46dd4b93f5d5883cb39)).

**Comment by haverma** [comment]
May not be high priority since batfish client in allinone mode is usable now with tracing enabled after batfish/batfish#1447
